_config.yml

```diff
@@ -16,12 +16,11 @@ url: https://docs.docker.com
 keep_files: ["v1.4", "v1.5", "v1.6", "v1.7", "v1.8", "v1.9", "v1.10", "v1.11", "v1.12", "v1.13", "v17.03", "v17.06", "v17.09", "v17.12", "v18.03"]
 exclude: ["_scripts", "apidocs/layouts", "Gemfile", "hooks"]
 
-# Component versions -- address like site.docker_ce_stable_version
+# Component versions -- address like site.docker_ce_version
 # You can't have - characters in these for non-YAML reasons
 
-latest_stable_docker_engine_api_version: "1.39"
-docker_ce_stable_version: "18.09"
-docker_ce_edge_version: "18.09"
+latest_engine_api_version: "1.39"
+docker_ce_version: "18.09"
 docker_ee_version: "18.09"
```

Comment:
FWIW; was thinking of combining the two variables above to just `engine_version` (given that these only hold the major version, and those are in sync now), but left that for a possible follow-up

Reply:
I like the idea of having one variable for Engine version.

Reply:
I'll do it in a follow-up; if we reserve this variable for the _major_ (`YY.MM`) version only, that should work; we may need to keep a separate variable if we want to keep track of patch releases possibly, but I don't think we currently do.

Reply:
We have page variables for DTR and UCP that represent _major.minor_ versioning, but I haven't seen an instance for Engine. We could double-check though, but I don't think it will be an issue.